core/src/main/java/org/lflang/TargetProperty.java

```diff
@@ -1012,6 +372,51 @@ public static TargetDecl extractTargetDecl(Target target, TargetConfig config) {
     return decl;
   }
 
+  private static KeyValuePair getKeyValuePair(
+      KeyValuePairs targetProperties, TargetProperty property) {
+    List<KeyValuePair> properties =
+        targetProperties.getPairs().stream()
+            .filter(pair -> pair.getName().equals(property.toString()))
+            .toList();
+    assert properties.size() <= 1;
+    return properties.size() > 0 ? properties.get(0) : null;
+  }
+
+  public static KeyValuePair getKeyValuePair(Model ast, TargetProperty property) {
```

Comment:
The `getKeyValuePair` methods shouldn't be used anymore, right?

Reply:
It is still used in some `validate` methods to look up key-value pairs of _other_ target properties (e.g., because they might be incompatible).